spec/ifBehaviors.js

```diff
@@ -1,31 +1,64 @@
+import {
+    arrayForEach
+} from 'tko.utils';
+
+import {
+    applyBindings, contextFor, dataFor
+} from 'tko.bind';
+
+import {
+    observable
+} from 'tko.observable';
+
+import {
+    Provider
+} from 'tko.provider';
+
+import {
+    options
+} from 'tko.utils';
+
+import * as templateBindings from '../index.js';
+
+import * as coreBindings from 'tko.binding.core'
+
+import '../node_modules/tko.utils/helpers/jasmine-13-helper.js';
+
 describe('Binding: If', function() {
     beforeEach(jasmine.prepareTestNode);
 
+    beforeEach(function(){
+        var provider = new Provider()
+        options.bindingProviderInstance = provider
+        bindingHandlers = provider.bindingHandlers
+        bindingHandlers.set(Object.assign(coreBindings.bindings, templateBindings.bindings));
+    });
+
     it('Should remove descendant nodes from the document (and not bind them) if the value is falsey', function() {
         testNode.innerHTML = "<div data-bind='if: someItem'><span data-bind='text: someItem.nonExistentChildProp'></span></div>";
         expect(testNode.childNodes[0].childNodes.length).toEqual(1);
-        ko.applyBindings({ someItem: null }, testNode);
+        applyBindings({ someItem: null }, testNode);
         expect(testNode.childNodes[0].childNodes.length).toEqual(0);
     });
 
-    it('Should leave descendant nodes in the document (and bind them) if the value is truthy, independently of the active template engine', function() {
-        this.after(function() { ko.setTemplateEngine(new ko.nativeTemplateEngine()); });
+    /*it('Should leave descendant nodes in the document (and bind them) if the value is truthy, independently of the active template engine', function() {
```

Comment:
The canonical way to [disable tests in Jasmine 1.3](http://jasmine.github.io/1.3/introduction.html#section-Disabling_Specs_and_Suites) is with `xit` or `xdescribe`, I believe.

We should also note in the conversation why we have disabled a test.